app/public/stylesheet/dashboard.css

```diff
@@ -0,0 +1,53 @@
+.dashboard_page{
+    margin: 0;
+    padding: 0;
+    background-color:rgb(250, 250, 250);
+    width:100%;
+    height:100%;
+    overflow: hidden;
+}
+
+.container{
+    width: 100% !important;
+}
+
+.inner_logo{
+    position: relative;
+    top: 1vh;
+}
+
+.sign_out_button{
+    color: white !important;
```

Comment:
Are you adding !important to these styles because you're overriding a previously establishedd styling? Or is it unnecessary?

Reply:
Has been fixed.